sos/report/plugins/ovn_central.py

```diff
@@ -25,6 +25,20 @@ class OVNCentral(Plugin):
     profiles = ('network', 'virt')
     containers = ('ovn-dbs-bundle.*', 'ovn_cluster_north_db_server')
 
+    def _find_ovn_controller_sock(self):
+        if self._container_name:
+            cmd = "find %s -name %s" % (self.ovn_controller_sock_path,
+                                        self.ovn_controller_sock_regex)
+            res = self.exec_cmd(cmd, timeout=None, foreground=True,
+                                container=self._container_name)
+            if res['status'] != 0:
+                self._log_error(
+                    "Could not retrieve ovn_controller socket path "
+                    "from container %s" % self._container_name
+                )
+                return
```

Comment:
Should this be `return ''` instead? Otherwise the `format()` below will convert the string to `ovn-appctl -t None connection-status`.

Alternatively, break out the connection-status command and gate it with a conditional.

Reply:
It is not really necessary, None will be appended to the output file name, but to make life easier for future parsing of the output file I will return the proposed _sfile in case we are not able to locate the real sock path.